### DISCUSSION TITLE
What's the difference with nordvpn docker image?

### DISCUSSION THREAD

**Discussion by jeremyfritzen** [bubuntux/nordlynx#122] [Q&A]
Hi!

I know that nordlynx docker image is the new VPN docker container project bubuntux wants to focus on.

But instead of using a new docker image (nordlynx), why not just setting the nordvpn `TECHNOLOGY` parameter to "nordlynx"?
What's the difference between nordvpn with "nordlynx" settings VS nordlynx image?

Thanks!

**Comment by imthenachoman** [comment]
Did you ever find out the difference? I can't tell?
